docs/user/ppl/cmd/stats.rst

```diff
@@ -238,6 +238,27 @@ Example::
     | 2.8613807855648994 |
     +--------------------+
 
+TAKE
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: TAKE(field [, size]). Return original values of a field.
+
+* field: mandatory. The field must be a text field.
+* size: optional integer. The number of values should be returned. Default is 10.
+
+Example::
+
+    os> source=accounts | stats take(firstname);
```

Comment:
Would you consider adding more examples with groupBy, size and may be patterns if possible?
or add a complex Example which covers all.
will you be including ITs in this PR or other PR?

Reply:
will add one with groupBy and ITs. I don't know if we should put `patterns` here since this is the stats doc

Reply:
added example 10 https://github.com/opensearch-project/sql/blob/2ad0f5eab9609cc6ced106a06ce7303ac396d425/docs/user/ppl/cmd/stats.rst#example-10-calculate-the-count-and-get-email-list-by-a-gender-and-span